boggle_blast.lic

```diff
@@ -0,0 +1,165 @@
+custom_require.call(%w[common common-healing common-items common-money common-travel drinfomon])
```

Comment:
💡 Please add a comment block with a link to elanthipedia (and some documentation for this script on elanthipedia)